samples/java-webtestclient/generated-docs/index.html

```diff
@@ -0,0 +1,2471 @@
+<td class="tableblock halign-left valign-top"><p class="tableblock">String</p></td>
+<td class="tableblock halign-left valign-top"><p class="tableblock">true<br>
+EN: false</p></td>
+<td class="tableblock halign-left valign-top"><p class="tableblock">Some information about the item.</p>
+<p class="tableblock">DE: Size must be between 2 and 10 inclusive.<br>
+EN: Size must be between 4 and 12 inclusive.<br>
+Length must be between 0 and 20 inclusive.</p></td>
+</tr>
+<tr>
+<td class="tableblock halign-left valign-top"><p class="tableblock">type</p></td>
+<td class="tableblock halign-left valign-top"><p class="tableblock">String</p></td>
+<td class="tableblock halign-left valign-top"><p class="tableblock">true</p></td>
+<td class="tableblock halign-left valign-top"><p class="tableblock">Country dependent type of the item.</p>
+<p class="tableblock">DE: Must be one of [klein, groß].<br>
+EN: Must be one of [small, big].<br>
+Size must be between 0 and 1000 inclusive.</p></td>
+</tr>
+</tbody>
+</table>
+</div>
+<div class="sect3">
+<h4 id="_response_fields_3"><a class="anchor" href="#_response_fields_3"></a><a class="link" href="#_response_fields_3">2.3.5. Response fields</a></h4>
+<div class="paragraph">
+<p>No response body.</p>
+</div>
+</div>
+<div class="sect3">
+<h4 id="_example_request"><a class="anchor" href="#_example_request"></a><a class="link" href="#_example_request">2.3.6. Example request</a></h4>
+<div class="listingblock">
+<div class="content">
+<pre class="highlightjs highlight"><code class="language-bash hljs" data-lang="bash">$ curl 'http://localhost:8080/items' -i -X POST \
+    -H 'Content-Type: application/json' \
+    -d '{"description":"Hot News","type":null}'</code></pre>
+</div>
+</div>
+</div>
+<div class="sect3">
+<h4 id="_example_response"><a class="anchor" href="#_example_response"></a><a class="link" href="#_example_response">2.3.7. Example response</a></h4>
+<div class="listingblock">
+<div class="content">
+<pre class="highlightjs highlight nowrap"><code class="language-http hljs" data-lang="http">HTTP/1.1 201 Created
+Location: http://localhost:8080/items/2
+Cache-Control: no-cache, no-store, max-age=0, must-revalidate
+Pragma: no-cache
+Expires: 0
+X-Content-Type-Options: nosniff
+X-Frame-Options: DENY
+X-XSS-Protection: 1 ; mode=block</code></pre>
+</div>
+</div>
+</div>
+</div>
+<div class="sect2">
+<h3 id="resources-item-resource-test-update-item"><a class="anchor" href="#resources-item-resource-test-update-item"></a><a class="link" href="#resources-item-resource-test-update-item">2.4. Update Item</a></h3>
+<div class="paragraph">
+<p><code>PUT /items/1</code></p>
+</div>
+<div class="paragraph">
+<p>Updates existing item.</p>
+</div>
+<div class="paragraph">
+<p>An example of returning an {@link HttpEntity}.</p>
+</div>
+<div class="sect3">
+<h4 id="_authorization_4"><a class="anchor" href="#_authorization_4"></a><a class="link" href="#_authorization_4">2.4.1. Authorization</a></h4>
+<div 
```

Comment:
URL is missing in docs if URI is set by UriBuilder.

Example:
```
webTestClient.get()
    .uri(uriBuilder -> uriBuilder.path("/items/search")
        .queryParam("desc", "main")
        .queryParam("hint", "1")
        .build())
    // ...
```
I will look at this.

Reply:
Fixed: 9bf28b0